api/column_family.cc

```diff
@@ -310,7 +310,7 @@ void set_column_family(http_context& ctx, routes& r) {
         return res;
     });
 
-    cf::get_column_family.set(r, [&ctx] (const_req req){
+    cf::get_column_family.set(r, [&ctx] (std::unique_ptr<request> req){
             vector<cf::column_family_info> res;
```

Comment:
I think this can be large too. Maybe it should be a chunked_vector.

Reply:
`sizeof(cf::column_family_info)` returns 224, so with 5000 tables it is in fact over 1MB. It can't be a chunked_vector though because column_family_info's move constructor isn't noexcept, but I think we can just use an std::list instead. Also, we can't use the `stream_object` method because it uses the json formatter from seastar which only accepts std vectors and maps, but I see that we have `stream_range_as_array` for that case.
Should I create a new PR for this change or is it OK to add it here?